doc/devel/documenting_mpl.rst

```diff
@@ -29,349 +29,170 @@ requirements that are needed to build the documentation.  They are listed in
+the ``*emphasis*`` role. This will keep Matplotlib's documentation consistent
+with Python's documentation:
 
-       A bit about :ref:`referring-to-mpl-docs`:
-          One more
+.. code-block:: rst
 
-  yields:
+  Here is a description of *argument*
 
-  .. seealso::
+Please do not use the ```default role```:
 
-     Using ResT :ref:`emacs-helpers`:
-        One example
 
-     A bit about :ref:`referring-to-mpl-docs`:
-        One more
+.. code-block:: rst
 
-* Please keep the :ref:`glossary` in mind when writing documentation. You can
-  create a references to a term in the glossary with the ``:term:`` role.
+   Please do not describe `argument` like this.
 
-* The autodoc extension will handle index entries for the API, but additional
-  entries in the index_ need to be explicitly added.
+nor the ````literal```` role:
 
-* Keyword arguments should be described using a definition list.
+.. code-block:: rst
 
-  .. note::
-     Matplotlib makes extensive use of keyword arguments as pass-through
-     arguments, there are a many cases where a table is used in place of a
-     definition list for autogenerated sections of docstrings.
+   Please do not describe ``argument`` like this.
 
 Figures
 =======
 
 Dynamically generated figures
 -----------------------------
 
-Figures can be automatically generated from scripts and included in
-the docs.  It is not necessary to explicitly save the figure in the
-script, this will be done automatically at build time to ensure that
-the code that is included runs and produces the advertised figure.
+Figures can be automatically generated from scripts and included in the docs.
+It is not necessary to explicitly save the figure from the script; this will be
+done automatically when the docs are built to ensure that the code that is
+included runs and produces the advertised figure.
 
-The path should be relative to the ``doc`` directory.  Any plots
-specific to the documentation should be added to the ``doc/mpl_examples/pyplots``
-directory and committed to git.  Plots from the ``examples`` directory
-may be referenced through the symlink ``mpl_examples`` in the ``doc``
-directory, e.g.:
+Plotting code should be added to the :file:`examples` directory.  Plots from
+the :file:`examples` directory may be referenced through the symlink
+``mpl_examples`` in the ``doc`` directory, e.g.:
 
 .. code-block:: rst
 
-  .. plot:: mpl_examples/pylab_examples/simple_plot.py
+  .. plot:: mpl_examples/lines_bars_and_markers/fill.py
 
-The ``:scale:`` directive rescales the image to some percentage of the
-original size, though we don't recommend using this in most cases
-since it is probably better to choose the correct figure size and dpi
-in Matplotlib and let it handle the scaling.
+To include plots in docstrings, see the following guide:
 
 Plot directive documentation
 ''''''''''''''''''''''''''''
 
 .. automodule:: matplotlib.sphinxext.plot_directive
+   :no-undoc-members:
 
 Examples
```

Comment:
As above.  This seems the heading (+ Tutorials) not "Figures" per se.  (I guess there are figures in the numpydocs?)